corehq/apps/locations/bulk_management.py

```diff
@@ -462,7 +462,7 @@ class LocationTreeValidator(object):
     """
     Validates the given type and location stubs
     """
-    def __init__(self, type_rows, location_rows, old_collection=None):
+    def __init__(self, type_rows, location_rows, old_collection):
```

Comment:
Might be useful to document these params. I definitely don't have an intuitive understanding of what they mean reading the names (especially `old_collection`).

Reply:
will do this in a separate PR

Reply:
9738674630d8a47b08933f462207c44da6823a3e

Reply:
thanks